wiki/Main_Page/id.md

```diff
@@ -1,25 +1,31 @@
 ---
 layout: main_page
-outdated: true
-outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
 ---
 
+<!-- Do not add any empty lines inside this div. -->
+
 <div class="wiki-main-page__blurb">
-Selamat datang di osu! wiki, sebuah proyek yang bertujuan untuk mendokumentasikan berbagai informasi yang terkait dengan osu!.
+Selamat datang ke osu! wiki, basis pengetahuan open-source yang berisi berbagai informasi terkait osu!, permainan ritme free-to-win.
 </div>
 
 <div class="wiki-main-page__panels">
 <div class="wiki-main-page-panel wiki-main-page-panel--full">
 
 # Pendahuluan
+  
+Artikel dan panduan penting yang akan membantu Anda mendalami osu!. Jika terjadi kesalahan atau hal-hal tidak jelas, pastikan untuk mengunjungi [Forum bantuan](https://osu.ppy.sh/forum/5).
```

Comment:
`Artikel-artikel berikut ini akan memandumu untuk dapat mengenal osu! lebih dalam. Apabila kamu memiliki pertanyaan-pertanyaan lebih lanjut, harap kunjungi [sub-forum Help](https://osu.ppy.sh/forum/5).`

nama sub-forum itu nggak diterjemahin btw :

![image](https://user-images.githubusercontent.com/36564236/130379544-fd6948b5-a791-4624-b528-a2e35d61271c.png)

Reply:
sip gotta fix this, as well to intro